README.md

```diff
@@ -40,34 +41,43 @@ With an api key that has GPT4 access run:
 
 - `export OPENAI_API_KEY=[your api key]`
 
+
 **Run**:
 
 - Create an empty folder. If inside the repo, you can run:
   - `cp -r projects/example/ projects/my-new-project`
-- Fill in the `main_prompt` file in your new folder
+- Fill in the `prompt` file in your new folder
 - `gpt-engineer projects/my-new-project`
   - (Note, `gpt-engineer --help` lets you see all available options. For example `--steps use_feedback` lets you improve/fix code in a project)
 
-**Results**:
+By running gpt-engineer you agree to our [ToS](https://github.com/AntonOsika/gpt-engineer/TERMS_OF_USE.md).
```

Comment:
This points to no file (I guess is a WIP, just surfacing it)

Reply:
This was not intended – blunder by me